docs/deployment/heroku.rst

```diff
@@ -52,3 +51,14 @@ Then log into your Heroku account, find the Heroku Scheduler addon in the active
 .. code-block:: bash
 
  python manage.py update_exchange_rates --all
+
+
+Enabling Elasticsearch
+----------------------
+
+By default, Saleor uses Postgres as a search backend, but if you want to switch to Elasticsearch, it can be easily achieved using the Bonsai plugin. In order to do that, run the following commands:
+
+.. code-block:: bash
```

Comment:
Let's update these to `.. code-block:: console` and include the `$` for prompt before commands.